pom.xml

```diff
@@ -32,6 +32,27 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.mapstruct</groupId>
+			<artifactId>mapstruct-processor</artifactId>
+			<version>1.5.5.Final</version>
+		</dependency>
+
+		<dependency>
+			<groupId>org.mapstruct</groupId>
+			<artifactId>mapstruct</artifactId>
+			<version>1.5.5.Final</version>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-data-jpa</artifactId>
+			<version>3.2.5</version>
```

Comment:
wersja tej biblioteki nie musi byc podana explciit -> ta biblioteka jest starterem springowym wiec mozemy ja wziac z worka dzieki czemu jak podbijemy wersje worka -> podbijemy wersje Springa to z automatu zaczyta nam rowniez odpowiednia wersje